src/tests/ftest/util/server_utils_base.py

```diff
@@ -177,10 +186,21 @@ def engine_params(self):
             list: a list of YamlParameters for each server engine
 
         """
-        engine_params = []
         if self.yaml is not None and hasattr(self.yaml, "engine_params"):
-            engine_params = self.yaml.engine_params
-        return engine_params
+            return self.yaml.engine_params
+        return []
+
+    @property
+    def control_metadata(self):
+        """Get the control plane metadata configuration parameters.
+
+        Returns:
+            ControlMetadataParameters: the control plane metadata configuration parameters or None
+                if not defined.
+        """
+        if self.yaml is not None and hasattr(self.yaml, "metadata_params"):
+            return self.yaml.metadata_params
+        return None
```

Comment:
Not sure I have a preference, but curious what you think?
```suggestion
        try:
            return self.yaml.metadata_params
        except AttributeError:
            return None
```